### PR TITLE
Missing `session_state` in generated token (#667)

### LINKED ISSUE #667: Missing `session_state` in generated token
The tokens generated after login are missing the `session_state` claim, which causes troubles on tenant: https://github.com/fabric8-services/fabric8-tenant/issues/638

related to https://github.com/fabric8-services/fabric8-auth/pull/618

### PR DESCRIPTION
**Pull request by xcoulon**
Fill the `session_state` claim of the generated access token
with a random UUID

Fixes #667

Signed-off-by: Xavier Coulon <xcoulon@redhat.com>

### CODE REVIEW

**Comment by fabric8cd** [comment]
@xcoulon snapshot fabric8-auth image is available for testing.  `docker pull docker.io/fabric8/fabric8-auth:SNAPSHOT-PR-668-2`

File: login/service.go [hunk 1]
```diff
@@ -636,44 +632,42 @@ func (keycloak *KeycloakOAuthProvider) CreateOrUpdateIdentityInDB(ctx context.Co
...
-				if err != nil {
-					return err
-				}
-				return nil
-			})
-		}
+	if identity.User.ID == uuid.Nil {
+		log.Error(ctx, map[string]interface{}{
+			"identity_id": identity.ID,
+		}, "Found Keycloak identity is not linked to any User")
```
Comment (sbose78) [inline login/service.go]: Technically, should not call it keycloak identity anymore :)

Comment (xcoulon) [reply login/service.go]: oh right, let me fix that ;)

**Comment by fabric8cd** [comment]
@xcoulon snapshot fabric8-auth image is available for testing.  `docker pull docker.io/fabric8/fabric8-auth:SNAPSHOT-PR-668-4`

**Pull request merged**